news/2017-07-22-beatmap-spotlight-june-2017.md

```diff
@@ -0,0 +1,108 @@
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/syncretism.jpg)](https://osu.ppy.sh/s/514144)
+
+It takes skill to make an interesting map out of a simple song, but **[RoseusJaeger](https://osu.ppy.sh/u/6709840)** has achieved that nicely here. Each difficulty features a variety of instrumental and vocal rhythm choices, layered together in a clean, fun and intuitive way, resulting in a beatmap set that is relaxing and fun to play for players of all skill levels. The creative use of 1/6 patterns adds some variety and character, helping this set to stand out among the many available choices in what has undoubtedly been a very productive month for osu!catch mapping.
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/moonlight-tango.jpg)](https://osu.ppy.sh/s/536001)
+
+Mapping lower BPM certainly brings its own challenges in osu!catch but **[JBHyperion](https://osu.ppy.sh/u/4879508)** has made it look near effortless here. Making excellent use of absolutely everything at his disposal, dashes, hyperdashes, anti-flow and some fairly unique stream patterns, it all comes together to represent the music fantastically whilst still being a real delight to play across all difficulties. Once again he delivers a level of quality every catch mapset should aspire to achieve.
+
+## osu!mania
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/illusion-of-inflict.jpg)](https://osu.ppy.sh/s/529574)
+
+This time, **[Reba](https://osu.ppy.sh/u/3087654)** is bringing us an almost perfectly structured beatmap in every aspect as well as seven guest difficulties. Although the BPM is not really high, you can find challenging patterns such as jacks, long notes, chordstreams and more, specially in **[Critical_Star](https://osu.ppy.sh/u/3793196)**'s. Talking about the hardest map in the set, it has everything mentioned before but way more dense than all the other diffculties as well as a few noticeable Slider Velocity changes at the last section of the map. Highly recommended to play since it has difficulties for every type of player, going from 1 star to almost 6 and a half.
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/exit-this-earth-atomosphere.jpg)](https://osu.ppy.sh/s/575053)
+
+It's always a good surprise to find an amazing mapset as a first ranked map, and this is the case with **[Protastic101](https://osu.ppy.sh/u/6712747)**'s set! This set shows a lot of diversity: Burst, Jumpstreams, long notes... But the most amazing point of the set is the usage of Slider Velocity, empathizing a lot of slower parts and creating a really powerful feeling on the harder ones. Overall, it creates a really challenging and technical map set, sprinkled with a lot rare patterns seldom seen in the ranked section.
```

Comment:
A triple dot is not really correct. Rather say `Burst, Jumpstreams, long notes - but the most amazing`